pages/agent/gcloud.md.erb

```diff
@@ -0,0 +1,183 @@
+# Running Buildkite Agent on Google Cloud Platform
```

Comment:
(I think I was matching the top-level title for AWS: "Running Buildkite Agent on AWS")